src/content/app/help/Help.tsx

```diff
@@ -78,10 +78,17 @@ const Help = () => {
     };
   }, []);
 
+  useEffect(() => {
+    if (pathname !== currentHistoryItem) {
+      dispatch(addPageToHistory(pathname));
+    } else {
+    }
```

Comment:
❓

Reply:
Forgot to remove this one. Added `no-empty` rule to ESLint to spot these here after.

Reply:
✅